src/sentry/migrations/0121_obliterate_group_inbox.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-11-02 20:25
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+BATCH_SIZE=100
+
+def obliterate_group_inbox(apps, schema_editor):
+    print("Running obliterate_group_inbox")
+    GroupInbox = apps.get_model("sentry.GroupInbox")
+    print("Obliterating GroupInbox. Count: ", GroupInbox.objects.all().count())
```

Comment:
I'd remove these print statements, they'll slow things down a lot with the full count.

Reply:
Updating based on our conversation: Just make the initial total count and subtract BATCH_SIZE each time

Reply:
Updated, will merge when CI finishes